File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/PeeringPeeringPeer.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class PeeringPeeringPeer {
    /// <summary>
    /// Gets or Sets Addrs
    /// </summary>
    [DataMember(Name="Addrs", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "Addrs")]
    public List<string> Addrs { get; set; }

    /// <summary>
    /// Gets or Sets Connected
    /// </summary>
    [DataMember(Name="Connected", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "Connected")]
    public bool? Connected { get; set; }

    /// <summary>
    /// Gets or Sets ID
    /// </summary>
    [DataMember(Name="ID", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ID")]
    public string ID { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class PeeringPeeringPeer {\n");
      sb.Append("  Addrs: ").Append(Addrs).Append("\n");
      sb.Append("  Connected: ").Append(Connected).Append("\n");
      sb.Append("  ID: ").Append(ID).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/peering_peering_peer.dart
part of swagger.api;

class PeeringPeeringPeer {
  
  List<String> addrs = [];

  bool connected = null;

  String ID = null;

  PeeringPeeringPeer();

  @override
  String toString() {
    return 'PeeringPeeringPeer[addrs=$addrs, connected=$connected, ID=$ID, ]';
  }

  PeeringPeeringPeer.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    addrs = (json['Addrs'] as List).map((item) => item as String).toList();
    connected = json['Connected'];
    ID = json['ID'];
  }

  Map<String, dynamic> toJson() {
    return {
      'Addrs': addrs,
      'Connected': connected,
      'ID': ID
     };
  }

  static List<PeeringPeeringPeer> listFromJson(List<dynamic> json) {
    return json == null ? new List<PeeringPeeringPeer>() : json.map((value) => new PeeringPeeringPeer.fromJson(value)).toList();
  }

  static Map<String, PeeringPeeringPeer> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, PeeringPeeringPeer>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new PeeringPeeringPeer.fromJson(value));
    }
    return map;
  }
}
